pallets/author-mapping/src/tests.rs

```diff
@@ -491,9 +480,7 @@ fn full_rotating_to_the_same_author_id_leaves_registration_in_tact() {
 			assert_noop!(
 				AuthorMapping::set_keys(
 					Origin::signed(1),
-					TestAuthor::Alice.into(),
-					TestAuthor::Alice.into(),
-					TestAuthor::Alice.into(),
+					(TestAuthor::Alice.into(), TestAuthor::Alice.into())
 				),
```

Comment:
I tend to like writing a test testing the migration itself. This is totally optional of course, but if you want some example, https://github.com/PureStake/moonbeam/blob/7a6f4bd158361173d76aabe99111a293182ac557/pallets/asset-manager/src/tests.rs#L447

Reply:
added the migration unit test `add_reverse_mapping_migration_works`